react/country/CZE.js

```diff
@@ -0,0 +1,147 @@
+import { POSTAL_CODE } from '../constants'
+
+export default {
+  country: 'CZE',
+  abbr: 'CZ',
+  postalCodeFrom: POSTAL_CODE,
+  postalCodeProtectedFields: ['state'],
+  fields: [
+    {
+      hidden: true,
+      name: 'country',
+      maxLength: 100,
+      label: 'country',
+      size: 'medium',
+    },
+    {
+      name: 'postalCode',
+      maxLength: 6,
+      label: 'postalCode',
+      required: true,
+      mask: '999 99',
+      regex: /^[1-7]\d{2}\ \d{2}$/,  //shares zipcodes with slovakia. Numbers 8,9,0 are reserved there. 
```

Comment:
is the space here intentional?
```suggestion
      regex: /^[1-7]\d{2}\d{2}$/,  //shares zipcodes with slovakia. Numbers 8,9,0 are reserved there. 
```

Reply:
According to the web, its common to place it.
https://en.wikipedia.org/wiki/List_of_postal_codes_in_the_Czech_Republic

Reply:
oh okay, nice